### PR TITLE
Update LIT5188MashafaMadbal.xml

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: 5001-6000/LIT5188MashafaMadbal.xml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5188MashafaMadbal" xml:lang="en" type="work">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
                 <title xml:lang="gez" xml:id="t1">መጽሐፈ፡ መድበል፡</title>
                 <title xml:lang="gez" corresp="#t1" type="normalized">Maṣḥafa Madbal</title>
-                
+
                 <title xml:lang="gez" xml:id="t2">ጉባዔ፡ ምሥጢራተ፡ ዘመናት፡</title>
                 <title xml:lang="gez" corresp="#t2" type="normalized">Gubāʿe Mǝśṭirata Zamanāt</title>
```
Comment (thea-m) [inline 5001-6000/LIT5188MashafaMadbal.xml]: Our guidelines only state the capitalising of the first word of titles, is there a particular reason for the capitalising throughout?

File: 5001-6000/LIT5188MashafaMadbal.xml [hunk 2]
```diff
@@ -53,22 +53,67 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                    <label xml:lang="en">Introduction to the second part</label>
+                  </div>
+                  <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Verses">
+                    <label xml:lang="en">Verses addresses to the Holy Trinity</label>
+                  </div>
+                  <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="FourGospels">
+                    <label xml:lang="en">The four Gospels in verses</label>
+                  </div>
+                  <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Testament">
+                    <label xml:lang="en"><foreign xml:lang="gez">መጽሐፈ፡ ኪዳን፡</foreign>in verses</label>
```
Comment (thea-m) [inline 5001-6000/LIT5188MashafaMadbal.xml]: I would use here a ref with the work IDs, or otherwise, transcribe the titles for better ease of reading

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

File: 5001-6000/LIT5188MashafaMadbal.xml (hunk 1 above)
Comment (DariaElagina) [reply 5001-6000/LIT5188MashafaMadbal.xml]: no, I just have forgotten to check the titles

File: 5001-6000/LIT5188MashafaMadbal.xml (hunk 2 above)
Comment (DariaElagina) [reply 5001-6000/LIT5188MashafaMadbal.xml]: The refs are not appropriate here, as the texts appear to be rather retelling of other texts

Comment (thea-m) [reply 5001-6000/LIT5188MashafaMadbal.xml]: (It does not matter in this case, but it is my understanding that the refs are used to point to the works in cases like this, they do not state the present of the works)

**Review by thea-m — APPROVED** [review]

**Pull request merged**